.github/bin/verify-concepts/verify-concepts.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+EXIT_CODE=0
+LANGUAGE_DIRS=$(find languages -mindepth 1 -maxdepth 1 -type d | sort)
+
+for LANGUAGE_DIR in $LANGUAGE_DIRS; do
+    CONFIG_JSON_FILE="$LANGUAGE_DIR/config.json"
+
+    # Skip track if it doesn't have a config.json file
+    if [[ ! -f "$CONFIG_JSON_FILE" ]]; then
+        continue
+    fi
+
+    LANGUAGE=$(jq -r ".language" $CONFIG_JSON_FILE)
+    CONCEPTS=$(jq -r ".exercises | .concept[] | .concepts[]" $CONFIG_JSON_FILE | sort -u)
+
+    for CONCEPT in $CONCEPTS; do
+        CONCEPT_DIR="$LANGUAGE_DIR/concepts/$CONCEPT"
+        CONCEPT_ABOUT_FILE="$CONCEPT_DIR/about.md"
+        CONCEPT_LINKS_FILE="$CONCEPT_DIR/links.json"
+
+		if [[ ! -f "$CONCEPT_ABOUT_FILE" ]]; then
```

Comment:
The indentation seems wrong here

Reply:
Ah, there were some tabs in there. I've converted them to spaces.